tools/Microsoft.Health.Dicom.SchemaManager/Microsoft.Health.Dicom.SchemaManager.csproj

```diff
@@ -0,0 +1,41 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Let's move these projects over to the `src` folder.

Reply:
Fixed in latest commit.